### PR TITLE
New design for game #16

### PR DESCRIPTION
**Pull request by KassandraA**
The task includes new UI and some bug fixes.

### CODE REVIEW

File: source/Tests/index-old.html [hunk 1]
```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
```
Comment (aannenko) [inline source/Tests/index-old.html]: According to its name, this file should not be included in this pull request and in the repo itself.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,7 @@
+index-v-01.html
+resize.html
+resize2.html
+style-old.css
```
Comment (aannenko) [inline .gitignore]: Please remove files from the repo and from the .gitignore that should not be in the repo.

File: source/Tests/view-old.js [hunk 3]
```diff
@@ -0,0 +1,454 @@
+/**
```
Comment (aannenko) [inline source/Tests/view-old.js]: According to the file's name, this file should not be included in this pull request as well as it should not exist in the repo.

File: source/css/style.css [hunk 4]
```diff
@@ -2,77 +2,546 @@
...
+    width: 30rem;
+    text-align: right;
+}
+#start_game > button,
+#settings_btn > button {
+    width: 30rem;
+    height: 30rem;
+    padding: 0;
+    border: none;
+    -webkit-border-radius: 50%;
```
Comment (aannenko) [inline source/css/style.css]: We've agreed to support only newest browser versions. Do we still need these specific styles despite this fact?

File: source/fonts/fontawesome-webfont.svg [hunk 5]
```diff
@@ -0,0 +1,2671 @@
+<?xml version="1.0" standalone="no"?>
```
Comment (aannenko) [inline source/fonts/fontawesome-webfont.svg]: Lots of new files are sent to this pull request. Can we make sure that we don't have unused files (fonts, styles, etc.) in the repo?

File: source/js/model.js [hunk 6]
```diff
@@ -390,6 +391,7 @@ function Game() {
     let _field = fieldSingleton.getInstance();
     let _winnerChecker = new WinnerChecker();
     let _iterator = new Iterator(getPlayersArray());
+    let movesCounter = 1;
```
Comment (aannenko) [inline source/js/model.js]: Please keep up with the coding style, use underscore before class-level variable names.

File: source/js/view-model.js [hunk 7]
```diff
@@ -27,7 +27,12 @@ function ViewModel() {
     };
 
     this.getPlayerName = function() {
-        return this.getCurrentStatus() === 'Tie' ? 'It' : _game.state.currentPlayerName;
+        // return this.getCurrentStatus() === 'Tie' ? 'It' : _game.state.currentPlayerName;
```
Comment (aannenko) [inline source/js/view-model.js]: We do not keep redundant code, especially if it is commented out. ;)

**Review by aannenko — CHANGES_REQUESTED** [review]
Several minor changes required and Edge support should be fixed. Aside from that - great job!

**Comment by KassandraA** [comment]
Changes are done.

File: source/css/style.css [hunk 8]
```diff
@@ -2,77 +2,496 @@
...
+}
+/*** Reset CSS end ***/
+
+
 /************* Main styles *************/
+html {
+    font-size: 10px;
+}
+body {
+    width: 100%;
```
Comment (aannenko) [inline source/css/style.css:37]: It appears, width setting can be removed.

**Review by aannenko — CHANGES_REQUESTED** [review]
Several minor things to consider:
- The scrollbar is too close to settings
- Shadows are truncated to the left and right from the controls on the Settings view
- Shadows should be more like on the mock, they should look more like a single background shadow for all elements
- The '+' button is thicker that the player rows
- Header's content moves up when the header is being resized
- The Settings button jumps left and right when you switch between Game and Settings views
- Colors are not accurate (blue, orange.)
- The gaps between cells do not match those on the mock
- In case of a narrow vertical browser window, both the Game and the Settings views shouls occupy the whole width of the window
- We have to refuse empty names for a player in the updatePlayer method of the model.js

**Comment by aannenko** [comment]
🥇

**Review by aannenko — APPROVED** [review]

**Pull request merged**